### ISSUE TITLE
tfjs-models/coco-ssd Broken Sample Code Provided (Node Module Requires Transpiling)

### ISSUE THREAD

**Issue by isaacanthony** [tensorflow/tfjs#3557]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version

@tensorflow-models/coco-ssd@2.0.4

#### Browser version

node v14.5.0

#### Describe the problem or feature request

The sample Node script on this page (https://github.com/tensorflow/tfjs-models/tree/master/coco-ssd#via-npm) doesn't work. To my eyes, it looks like the @tensorflow-models/coco-ssd node_module has to be transpiled before use. My understanding is that shouldn't be a requirement of the downstream app.

#### Code to reproduce the bug / link to feature request
If you would like to get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

GitHub issues for this repository are tracked in the [tfjs union repository](https://github.com/tensorflow/tfjs/issues).

Please file your issue there, following the guidance in [that issue template](https://github.com/tensorflow/tfjs/blob/master/ISSUE_TEMPLATE.md).

Here is the code I ran:

**Dockerfile**
```
FROM node:latest
RUN npm install @tensorflow-models/coco-ssd@2.0.4
COPY main.js /main.js
ENTRYPOINT ["node", "/main.js"]
```

**main.js**
```
const cocoSsd = require('@tensorflow-models/coco-ssd');

(async () => {
  const img = document.getElementById('img');

  // Load the model.
  const model = await cocoSsd.load();

  // Classify the image.
  const predictions = await model.detect(img);

  console.log('Predictions: ');
  console.log(predictions);
})();
```

The above code matches exactly the dummy example provided in the README (https://github.com/tensorflow/tfjs-models/tree/master/coco-ssd#via-npm), except I had to wrap everything in an async function to be able to use the awaits.

Here is the error message:
```
/node_modules/@tensorflow-models/coco-ssd/dist/index.js:17
import * as tfconv from '@tensorflow/tfjs-converter';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:1117:16)
    at Module._compile (internal/modules/cjs/loader.js:1165:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
    at Module.load (internal/modules/cjs/loader.js:1050:32)
    at Function.Module._load (internal/modules/cjs/loader.js:938:14)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (/main.js:1:17)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
```

To my eyes, this looks like a transpiling issue with @tensorflow-models/coco-ssd using ES6 imports. I have spent some time referencing the dummy example to try to get babel to work to transpile my @tensorflow-models/coco-ssd node_module, but to my understanding, this should not be a requirement of using the module. From what I have read, a simple fix may be to just add `"type": "module"` to the @tensorflow-models/coco-ssd package.json.

Let me know if you need any more information. Thanks!

PS: An aside, but the link on this npmjs page for the "demo app" is also broken: https://www.npmjs.com/package/@tensorflow-models/coco-ssd#via-npm. It takes me to a GitHub 404.

**Comment by pyu10055** [comment]
This has been fixed, please give the new release a try 2.1.0

**Comment by isaacanthony** [comment]
That version works for me! Thank you for your quick turnaround.
